rss-client/src/app/Cart/cart.module.ts

```diff
@@ -24,7 +24,7 @@ import { QuizModule } from '../Quiz/quiz.module';
     FormsModule,
     ReactiveFormsModule,
     AppRoutingModule,
-    MatDialogModule,
+  //  MatDialogModule,
```

Comment:
why are we commenting out instead of deleting?

Reply:
removed comments